tendermint/src/abci/code.rs

```diff
@@ -19,17 +19,17 @@ pub enum Code {
 }
 
 impl Default for Code {
-    fn default() -> Code {
```

Comment:
So in general I think this is probably good, but I wonder if in the case of large `impl` blocks if it would be better to use the explicit type name.